teleport-node/update-necocli.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/sh -ex
+
+export HTTPS_PROXY="http://squid.internet-egress.svc:3128"
+
+DATE=$(curl -s "https://api.github.com/repos/cybozu-go/neco/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")' | sed -e "s/release-//")
+FILE="neco-operation-cli-linux_${DATE}_amd64.deb"
+
+CLI_PATH="${HOME}/neco-operation-cli/usr/bin"
+if [ ! -f "${HOME}/.profile" ]  || [ ! "$(cat ${HOME}/.profile | grep ${CLI_PATH})" ]; then
+    echo "export PATH=${PATH}:${HOME}/neco-operation-cli/usr/bin" >> ${HOME}/.profile
+fi
+
+if [ ! -f $FILE ]; then
+    echo "Downloading and extracting Neco CLI tools..."
+    curl -sLf -O https://github.com/cybozu-go/neco/releases/download/release-${DATE}/neco-operation-cli-linux_${DATE}_amd64.deb
```

Comment:
Let's use the FILE valiable.
```suggestion
    curl -sLf -O https://github.com/cybozu-go/neco/releases/download/release-${DATE}/${FILE}
```